tests/filters/test_monster_filter.py

```diff
@@ -41,6 +41,27 @@ def test_monster_id(self):
         for e in [fail1, fail2, fail3]:
             self.assertFalse(mon_filter.check_event(e))
 
+    def test_rarity_id(self):
+        # Create the filters
+        settings = {"rarity": ["common", "Very Rare"]}
+        mon_filter = Filters.MonFilter('filter1', settings)
+
+        # Generate events that should pass
+        pass1 = Events.MonEvent(generate_monster({"rarity": 0}))
+        pass2 = Events.MonEvent(generate_monster({"rarity": 3}))
+        # Test passing events
+        for e in [pass1, pass2]:
+            self.assertTrue(mon_filter.check_event(e))
+
+        # Generate events that should fail
+        fail1 = Events.MonEvent(generate_monster({"rarity": 1}))
+        fail2 = Events.MonEvent(generate_monster({"rarity": 2}))
+        fail3 = Events.MonEvent(generate_monster({"rarity": 4}))
+
+        # Test failing events
+        for e in [fail1, fail2, fail3]:
+            self.assertFalse(mon_filter.check_event(e))
+
```

Comment:
why not use the shortened method filter checks to reduce the space required and present a cleaner code appearance?

Reply:
Which filter checks are we talking about? This looks the same as the other tests to me

Reply:
```python
     def test_cp(self):
         settings = {'min_cp': 20, 'max_cp': 500}
         mon_filter = Filters.MonFilter('cp_filter', settings)
         for i in [20, 250, 500]:
             self.assertTrue(mon_filter.check_event(create_event({'cp': i})))
 
         for i in [19, 501, 9999]:
             self.assertFalse(mon_filter.check_event(create_event({'cp': i})))
```

Reply:
Fixed in https://github.com/PokeAlarm/PokeAlarm/commit/9bfb0bcaadbd67c37fc20bacec8e187435e7a220.